docs/source/development/add-ons/test-add-ons-18.md

```diff
@@ -0,0 +1,12 @@
+---
+myst:
+  html_meta:
+    "description": "Test add-ons in Volto 18"
+    "property=og:description": "Test add-ons in Volto 18"
+    "property=og:title": "Test add-ons in Volto 18"
+    "keywords": "Volto, Plone, testing, test, CI, add-ons"
+---
+
+# Test add-ons in Volto 18
+
+For testing add-ons in Volto 18, see {doc}`../../contributing/testing`.
```

Comment:
This refers to testing Volto core.

Better mention in short the make commands for testing here in this chapter.

Reply:
Fixed in 9ce649802